src/Compilers/CSharp/Test/Semantic/Semantics/ExpressionBodiedMemberTests.cs

```diff
@@ -382,5 +382,354 @@ class Program
                 //     private void M() => (new object());
                 Diagnostic(ErrorCode.ERR_IllegalStatement, "(new object())").WithLocation(4, 25));
         }
+
+        [Fact, WorkItem(1702, "https://github.com/dotnet/roslyn/issues/1702")]
```

Comment:
My sense is that the set of tests is overkill (i.e. not worth the additional execution time every time tests are run, relative to the probability of catching problems). I recommend combining them into a single test case (i.e. a single compilation).